src/layouts/Docs.tsx

```diff
@@ -80,47 +81,33 @@ const ContentContainer = (props: ContentContainerProps) => (
   />
 )
 
-const baseHeadingStyle: HeadingProps = {
-  fontFamily: "mono",
-  textTransform: "uppercase",
-  fontWeight: "bold",
-  scrollMarginTop: 40,
-}
+const baseHeadingClasses = "font-mono uppercase font-bold scroll-mt-40"
 
-const H1 = (props: HeadingProps) => (
+const H1 = (props: HTMLAttributes<HTMLHeadingElement>) => (
   <MdHeading1
-    {...baseHeadingStyle}
-    fontSize={{ base: "2rem", md: "2.5rem" }}
-    mt={{ base: 0, md: 8 }}
-    mb="8"
+    className={cn(baseHeadingClasses, "max-md:mt-0 max-md:text-[2rem]")}
     {...props}
   />
 )
 
-const H2 = (props: HeadingProps) => (
+const H2 = (props: HTMLAttributes<HTMLHeadingElement>) => (
   <MdHeading2
-    {...baseHeadingStyle}
-    fontSize="2xl"
-    lineHeight={{ base: 1.2, md: 1.4 }}
-    pb={2}
-    mt={12}
-    borderBottom="1px"
-    borderColor="border"
+    className={cn(
+      baseHeadingClasses,
+      "mt-12 border-b-[1px] border-[#e5e5e5] pb-2 text-2xl max-md:leading-4xs dark:border-[#333]"
```

Comment:
This approach is taken because before: `{ base: 1.2, md: 1.4 }`, where the line-height of `1.4` is already defined in `commonHeadingAttributes()` in `MdComponents`. So using `max-md` ensures we only need to redefine the line-height at the smaller screen-sizes without having to explicitly redeclare `1.4` for the larger sizes, as `H2` here is the only one that originally defined with a smaller line-height at the smaller screen widths.

Reply:
Got it, forgot that we define these at a higher level in the tree. I think it is leaving this line height for now to keep this PR small.